libvirt/tests/src/virtual_network/iface_bridge.py

```
@@ -123,7 +125,7 @@ def check_net_functions(guest_ip, ping_count, ping_timeout, guest_session, host_
     iface_script_bk = os.path.join(data_dir.get_tmp_dir(), "iface-%s.bk" % iface_name)
     attach_interface = "yes" == params.get("attach_interface", "no")
     iface_model = params.get("iface_model", "virtio")
-    iface_source = eval(params.get("iface_source", "{'bridge':'test_br0'}"))
+    iface_source = eval(params.get("iface_source", "{'bridge':'%s'}" % bridge_name))
```

Ruff (0.15.2)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)